mailpoet/lib/Config/Menu.php

```diff
@@ -413,7 +430,8 @@ private function registerMailPoetMenu() {
       $this->wp->addSubmenuPage(
         self::MAIN_PAGE_SLUG,
         $this->setPageTitle('Automation'),
-        'Automation',
+        // @ToDo Remove Beta once Automation is no longer beta.
+        '<span>Automation</span><span class="mailpoet-beta-badge">Beta</a>',
```

Comment:
Could you pls make the `Automation` string translatable, @websupporter? I won't do this in my branch, so we don't create conflicts.

Additionally, we can ask @NeosinneR if the menu item should be `Automation` or `Automations` and where in the menu (which order) it should lie.

Reply:
Let's go with Automations, as all other menu items are written in plural. 

As for translations - are other items translated? If so, I'd also translate this one.

Reply:
Menu items: Some where translateable, others not. I made them all translateable in [fc80f73](https://github.com/mailpoet/mailpoet/pull/4448/commits/fc80f73ea27c55f189b9b0374169d50e43b56e6b)

The Automations menu item has now been moved [db8fc2d](https://github.com/mailpoet/mailpoet/pull/4448/commits/db8fc2d9b55fd036ecfef5ad5784eb707afe0154) and renamed [d4259fa](https://github.com/mailpoet/mailpoet/pull/4448/commits/d4259fa5d9b364e12a7ac2e9595477ea457c6f19)